extensions/src/org/pentaho/platform/web/http/api/resources/services/FileService.java

```diff
@@ -342,6 +342,8 @@ public void doMoveFiles( String destPathId, String params ) throws FileNotFoundE
       }
     } catch ( IllegalArgumentException e ) {
       throw e;
+    } catch ( UnifiedRepositoryAccessDeniedException e ) {
+      throw new IllegalArgumentException();
```

Comment:
You should pass on the original exception here - this is swallowing what the actual error was.